custom_images/redis-3.2/README.md

```diff
@@ -0,0 +1,8 @@
+# Redis 3.2 Custom Images
+
+This directory contains the three custom images used to deploy the Redis 3x-HA
```

Comment:
Just a nit, but these images aren't specific to the 3x-ha plan (which will need to get renamed anyway).